### ISSUE TITLE
I set the seed, but the output is still inconsistent

### ISSUE THREAD

**Issue by wuxi-dixi** [openai/openai-cookbook#877]
I set up a seed by referring to the following website, but the output is still inconsistent. I tried it many times and didn't get consistent output.

https://cookbook.openai.com/examples/deterministic_outputs_with_the_seed_parameter

![截屏2023-11-22 09 55 04](https://github.com/openai/openai-cookbook/assets/85912764/375bc8ad-f3fd-4e36-9e9e-dff4081ae23c)
![截屏2023-11-22 09 55 10](https://github.com/openai/openai-cookbook/assets/85912764/377493ff-632a-4af7-9d51-826ccfcb5934)

**Comment by ZahrizhalAli** [comment]
I think the temperature set in this code is quite high:
`response = openai.chat.completions.create(
            model=GPT_MODEL,
            messages=messages,
            seed=seed,
            max_tokens=200,
            temperature=0.7,
        )`
Accorind to documentation here : https://platform.openai.com/docs/api-reference/audio/createTranscription , higher temperature causes the model to be more random, whereas lower i.e temperature=0.2 makes the model more deterministic.

**Comment by eleanorr125** [comment]
If you're experiencing inconsistent outputs despite setting a seed in your code, there are a few things you can check and adjust:

Library Versions:
Ensure that you are using the same versions of the libraries as the code example. Different library versions can behave differently.

**Environment:**
Confirm that your programming environment is consistent. If you're working in a Jupyter [notebook []or](https://scholarden.com/) a similar environment, make sure that all cells are executed in order.

**Seed Placement:**
Verify that you are setting the seed at the appropriate place in your code. For example, if you're using a deep learning framework like TensorFlow or PyTorch, set the seed right after importing the library and before building your model.

Global vs. Local Seed:
Check if the library you are using requires setting the seed globally or locally for specific operations. Some libraries have multiple random number generators, and setting the seed at different levels might be necessary for consistency.

**Hardware Acceleration:**
If you're using GPU acceleration, randomness can be affected. You might need to ensure that GPU operations are also seeded properly if applicable.

**Pseudo-Random Number Generator (PRNG):**
Understand the underlying PRNG used by the library. If it's a complex application with multiple sources of randomness, setting the seed might not guarantee complete determinism.

**Reproducibility Flags:**
Some libraries may have additional flags or settings for reproducibility. Check the documentation for any specific flags that need to be set.

**Debugging Output:**
Print out intermediate values or randomness sources during your code execution to identify where the inconsistency is coming from.

**Comment by artkpv** [comment]
Are current models that support seed only the same as 2 years ago? Namely,  gpt-4-1106-preview and gpt-3.5-turbo-1106. Thanks
